### PR TITLE
[Zone] Winterspring

### PR DESCRIPTION
**Pull request by Knindzagxg**
Winterspring is now fully updated, since every creature in zone was missing SmartAI.

Working on every zone at the moment, and will post updates once i'm done with each.

To test it:
Apply SQL 
Restart your server

You can manually spawn NPC by ID to check SmartAI or you can go to zone and check like that as well.

.npc add ID
or
.tele Winterspring and run / test around.

### CODE REVIEW

**Comment by Geekena** [comment]
damage and spells are ok
no problems aswell

File: data/sql/updates/pending_db_world/rev_1589121527805378200.sql [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+(7431, 0, 0, 0, 0, 0, 100, 0, 1500, 2000, 10000, 11000, 11, 24331, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Frostsaber - In Combat - Cast \'24331\'');
+
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 7430;
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 7430);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(7430, 0, 0, 0, 0, 0, 100, 0, 1500, 2000, 10000, 11000, 11, 24331, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Frostsaber Cub - In Combat - Cast \'24331\'');
+
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 10738;
```
Comment (Helias) [inline data/sql/updates/pending_db_world/rev_1589121527805378200.sql:37]: I think that you put the wrong entry here `10738`, instead of `10199`

Comment (Knindzagxg) [reply data/sql/updates/pending_db_world/rev_1589121527805378200.sql:37]: It should be 10738 for Warchief, dunno how i typed 10199. (https://wotlkdb.com/?npc=10738)
Nice find mate.

Comment (Knindzagxg) [reply data/sql/updates/pending_db_world/rev_1589121527805378200.sql:37]: Grizzle have same AI as well, that's the problem oO
Gonna add it now as well.

Comment (Knindzagxg) [reply data/sql/updates/pending_db_world/rev_1589121527805378200.sql:37]: Edit: Will add for Grizzle in Classic Rare AI, so this should be good for now.
All sorted.

**Comment by Kitzunu** [comment]
Ty for the PR

**Pull request merged**